src/FTPPath.cpp

```diff
@@ -16,13 +16,21 @@ FTPPath::~FTPPath()
 
 void FTPPath::changePath(String path)
 {
-	std::list<String> p = splitPath(path);
-	std::copy(p.begin(), p.end(), std::back_inserter(_Path));
+if(*path.begin() == '/') // Added Akoro 2021-02-27
```

Comment:
please use `path.StartsWith()`: https://www.arduino.cc/reference/en/language/variables/data-types/string/functions/startswith/

Reply:
It is don't need. I have returned to originnal code